src/Uno.UWP/UI/Input/GestureRecognizer.cs

```diff
@@ -135,14 +135,8 @@ internal void ProcessBeforeUpEvent(PointerPoint value, bool isRelevant)
 
 		internal void ProcessUpEvent(PointerPoint value, bool isRelevant)
 		{
-#if IS_UNIT_TESTS || UNO_REFERENCE_API
-			if (_gestures.TryGetValue(value.PointerId, out var gesture))
-			{
-				_gestures.Remove(value.PointerId);
-#else
 			if (_gestures.Remove(value.PointerId, out var gesture))
```

Comment:
This overload wasn't available in `netstandard2.0` and .NET Framework. So I think the conditional code was needed when we were building reference against netstandard2.0 and unit tests against net461. This is no longer necessary, and it's probably faster now.